lottie/src/main/java/com/airbnb/lottie/network/LottieFetchResult.java

```diff
@@ -0,0 +1,20 @@
+package com.airbnb.lottie.network;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStream;
+
+import androidx.annotation.Nullable;
+
+public interface LottieFetchResult extends Closeable {
```

Comment:
Please add a javadoc if you are adding a new public api

Reply:
done